integrations/elasticsearch/tests/test_document_store.py

```diff
@@ -242,16 +241,16 @@ def test_bm25_not_all_terms_must_match(self, document_store: ElasticsearchDocume
         Test that not all terms must mandatorily match for BM25 retrieval to return a result.
         """
         documents = [
-            Document(id=1, content="There are over 7,000 languages spoken around the world today."),
+            Document(id="1", content="There are over 7,000 languages spoken around the world today."),
```

Comment:
Did you switched to strings here for consistency with other tests?

Reply:
The `Document.id` should be a str and not an int

https://github.com/deepset-ai/haystack/blob/main/haystack/dataclasses/document.py#L71

Reply:
Got it. It's just weird that the test still passes using ints.